public/js/home.js

```diff
@@ -0,0 +1,9 @@
+(function init() {
+    slider(
+      document.querySelector('.prev'),

```

Comment:
Why select each element like that, its a search in whole page!! From this point you could just pass the parent wrapper of carousel and rest of the elements can be found inside it.